Update nl-NL.coffee

JurianLock opened the pull request:
Some general spelling checks and an update for (nearly) 100% translation.

Imperadeiro98 commented on app/locale/nl-NL.coffee:
```diff
@@ -551,11 +551,11 @@ module.exports = nativeDescription: "Nederlands (Nederland)", englishDescription
-    feature1: "meer dan $t(data.levelsCount) basislevels over $t(data.worldsCount) werelden" # {change}
+    feature1: "+$t(data.levelsCount) basislevels over $t(data.worldsCount) werelden" # {change}
     feature2: "$t(data.heroesCount) sterke <strong>nieuwe helden</strong> met unieke vaardigheden!" # {change}
     feature3: "$t(data.bonusLevelsCount) bonuslevels" # {change}
```
Can you just update these strings to match the English version? Otherwise, the numbers won't show up.

JurianLock replied:
I will.

Imperadeiro98 replied:
Can I merge this PR or will you update it?

JurianLock replied:
Just changed the + sign back to "meer dan". Does this fix it?

Imperadeiro98 commented:
I guess I didn't explain myself correctly. What I meant was that in the [English file](https://github.com/codecombat/codecombat/blob/master/app/locale/en.coffee#L554-L556), the variables are written differently. For example, $t(data.levelsCount) should become \__levelsCount\__, and the same for the other 3 variables.

Sorry for not explaining correctly the first time.

JurianLock commented:
Oh, now I understand. I'll make a new update ;)

JurianLock commented:
- Changed the variable references to match the English ones --> from t$(x) to _x_
- Added a some words.
- Changed 1 or 2 Dollar signs to Euro signs (not variables).

Imperadeiro98 commented on app/locale/nl-NL.coffee:
```diff
@@ -551,11 +551,11 @@ module.exports = nativeDescription: "Nederlands (Nederland)", englishDescription
-    feature3: "$t(data.bonusLevelsCount) bonuslevels" # {change}
+    feature1: "meer dan _data.levelsCount_ basislevels over _data.worldsCount_ werelden" # {change}
+    feature2: "_data.heroesCount_ sterke <strong>nieuwe helden</strong> met unieke vaardigheden!" # {change}
+    feature3: "_data.bonusLevelsCount_ bonuslevels" # {change}
```
Only one thing left: you should take the 'data.' part out and add an extra underscore so that there are two underscores before and after the word (like in the English version).

@nwinter, can't this conversion be made with a script?

JurianLock replied:
Woops, I should've read that more thoroughly. I'm on it.

nwinter replied:
Good point, I updated them automatically for all languages–thanks @Imperadeiro98.
